### PR TITLE
Modify index page markup to fix elementary/website issue #1526

### PR DESCRIPTION
**Pull request by Tardog**
Change markup on the developer site to fix https://github.com/elementary/website/issues/1526

Please see https://github.com/elementary/website/pull/1564 for the CSS fixes. https://github.com/elementary/website/pull/1564

### CODE REVIEW

File: src/houston/views/index.pug [hunk 1]
```diff
@@ -2,7 +2,7 @@ extends /layout/main
 
 block head
   link(rel="stylesheet", type="text/css", media="all", href="/styles/index.css")
-  link(rel="stylesheet", type="text/css", media="all", href="https://elementary.io/styles/solarized_light.css")
+  link(rel="stylesheet", type="text/css", media="all", href="http://localhost:8000/styles/solarized_light.css")
```
Comment (danrabbit) [inline src/houston/views/index.pug]: Oops, you probably didn't mean to submit this :D

**Comment by Tardog** [comment]
Duh. Sorry for missing that. Links are back to normal now.

File: src/houston/views/index.pug [hunk 2]
```diff
@@ -41,38 +41,42 @@ block body
 
   div.grid: div.two-thirds
     img(src='https://elementary.io/images/icons/places/128/distributor-logo.svg')
-    h1 Build for 
+    h1 Build for
```
Comment (btkostner) [inline src/houston/views/index.pug]: The space is actually needed here so the text shows up correctly.

Comment (Tardog) [inline src/houston/views/index.pug]: My editor is configured to trim trailing spaces, so that’s where they went. It’s fixed now.

**Pull request merged**